iogt/templates/polls/detail.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
```

Comment:
all the templates in `polls` should be removed, and referenced from the molo.polls app.